wasm.yaml

```
@@ -1142,6 +1173,31 @@ pages:
         ```
 
         Underneath the covers we take advantage of JavaScript's `eval` function to make these functions real.
+    fr:
+      title: Déclaration de Fonctions
+      content_markdown: |
+      Pour déclarer une fonction, nous avons besoin de deux choses:
```

indentation issue (code highlighting was confused)
```suggestion
          Pour déclarer une fonction, nous avons besoin de deux choses:
```